components/style/values/specified/counters.rs

```diff
@@ -82,7 +82,7 @@ type CounterStyleType = CounterStyleOrNone;
 /// The specified value for the `content` property.
 ///
 /// https://drafts.csswg.org/css-content/#propdef-content
-#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue)]
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToComputedValue, ToCss)]
 pub enum Content {
```

Comment:
I think the only thing missing here is `#[css(iterable)]` right before the `Items` variant.

Reply:
oh, alright. thanks :)